lib/vm_writer.rb

```diff
@@ -33,4 +33,8 @@ def write_call(name, num_args)
   def write_arithmetic(command)
     @output.puts "add"
   end
+
+  def write_return
+    @output.print "return"
```

Comment:
In the expected file we're testing against, there is no newline just EOF. To keep it consistent, I'll add a newline to the test file.

Reply:
OK, cool. (The reason it didn’t have a newline is because of your editor configuration, I think — `tools/JackCompiler.sh` does emit a trailing newline.)